src/clr-base/tsconfig.lib.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "../../out-tsc/lib",
+    "target": "es2015",
```

Comment:
Previously our target had to be `es5` for IE support. Is that no longer the case?

Reply:
Library configs will always produce a es5 bundle, see the output `dist/bundles/*umd.js`